crates/viewer/re_space_view_text_document/Cargo.toml

```diff
@@ -10,7 +10,7 @@ readme = "README.md"
 repository.workspace = true
 rust-version.workspace = true
 version.workspace = true
-include = ["../../LICENSE-APACHE", "../../LICENSE-MIT", "**/*.rs", "Cargo.toml"]
+include.workspace = true
```

Comment:
The main change in this PR is that we always include `Cargo.lock` in the crate package zip. That doesn't really do anything, unfortunately, but at least it is now consistent.